Mathlib/Topology/MetricSpace/Pseudo/Defs.lean

```diff
@@ -1013,6 +1013,23 @@ theorem isOpen_ball : IsOpen (ball x ε) :=
   isOpen_iff.2 fun _ => exists_ball_subset_ball
 #align metric.is_open_ball Metric.isOpen_ball
 
+lemma isClosed_closedBall (x : α) (r : ℝ) :
```

Comment:
I think this is precisely `isClosed_ball` in the file `Topology.MetricSpace.Pseudo.Lemmas`.

Reply:
Changed import